sentry_sdk/integrations/gcp.py

```diff
@@ -143,6 +153,18 @@ def event_processor(event, hint):
 
         request["url"] = "gcp:///{}".format(environ.get("FUNCTION_NAME"))
 
+        if hasattr(gcp_event, "method"):
```

Comment:
@untitaker I've modified event processor for GCP integration to get request related for Sentry dashboard.